code/__HELPERS/lists.dm

```diff
@@ -855,3 +855,23 @@
 			return FALSE
 
 	return TRUE
+
+// Pick something else from a list than we last picked
+/proc/pick_different_from_list(list/l, current_index)
+	var/next_index
+
+	if(!islist(l))
+		return FALSE
```

Comment:
```suggestion
		return null
```
Since we aren't sending anything back. Yes they both evaluate to false but this makes more readability sense.

Reply:
This is good to know